src/jaegertracing/SpanContext.h

```diff
@@ -164,6 +164,12 @@ class SpanContext : public opentracing::SpanContext {
         forEachBaggageItem(f);
     }
 
+    std::unique_ptr<opentracing::SpanContext> Clone() const noexcept
+    {
```

Comment:
Can you add a lock guard here please:

`std::lock_guard<std::mutex> lock(_mutex);`